logp/core.go

```diff
@@ -267,7 +267,8 @@ func loadLogger() *coreLogger {
 
 func storeLogger(l *coreLogger) {
 	if old := loadLogger(); old != nil {
-		old.rootLogger.Sync()
+		// ignore the error
+		_ = old.rootLogger.Sync()
```

Comment:
That's the reason I'm not a huge fan of this `errcheck`lint... But it's not a strong opinion.

Anyways I believe the `// ignore the error` comment is redundant here, the error is clearly being ignored.